data/interfaces.txt

```diff
@@ -261,6 +270,12 @@ interface "preferences"
 
 
 
+interface "mini-map" top
```

Comment:
Would it make more sense for this to be part of the "hud" interface, so people could customize it along with the location for the messages, ammo, escorts, etc.?

Reply:
It could be placed there, yes, though I think both locations are equally customizable. Placing it inside the "hud" interface does help imply to anyone doing ui tweaks when the minimap is used.

I'll make the changes shortly.